snappy/click.go

```diff
@@ -316,12 +326,12 @@ func (m *packageYaml) addPackageServices(baseDir string, inhibitHooks bool, inte
 		}
 		// If necessary, generate the DBus policy file so the framework
 		// service is allowed to start
-		if m.Type == snap.TypeFramework && service.BusName != "" {
-			content, err := genBusPolicyFile(service.BusName)
+		if m.Type == snap.TypeFramework && app.BusName != "" {
+			content, err := genBusPolicyFile(app.BusName)
```

Comment:
Indeed, sorry for that. Is https://github.com/mvo5/snappy/compare/feature/snap-yaml3...mvo5:feature/snap-yaml-refactor-type?expand=1 a start in the right direction?